### DISCUSSION TITLE
被检测到使用第三方客户端而冻结账户

### DISCUSSION THREAD

**Discussion by aimiku** [Mrs4s/go-cqhttp#1657] [General]
看了一下 默认device.json里面product是mirai。请问和这个有关系吗？
![E1C1598311AEC5BEA9DCDC836CCD10B2](https://user-images.githubusercontent.com/10482479/187350859-e3473362-e1e2-4ad7-a4ff-0c39362e8435.png)
![75557B41D3B194F184DA07E0C5967755](https://user-images.githubusercontent.com/10482479/187350868-024ffd52-7b44-4d21-ac52-46c38285f3c6.png)

**Comment by aimiku** [comment]
已经因为第三方客户端登录被冻结3次了。。有人遇到过类似的问题吗？

**Reply by clacknya** [reply]
以前遇到过，那个号已经没了
